README.md

```diff
@@ -4,23 +4,50 @@ This project is a port of the ownCloud documentation, that was previously genera
 Fundamentally, not that much has changed.
 
 All of the same information is still available.
-However, here’s what has changed:
+However, here's what has changed:
 
 1. The platform (and tools) used to build the documentation, which is [Antora](./docs/what-is-antora.md).
 2. The file format that the documentation is written in, which is [AsciiDoc](./docs/what-is-asciidoc.md).
 3. The <abbr title="User Interface">UI</abbr> & <abbr title="User Experience">UX</abbr> of the documentation
 
+## Quick Start Guide
+
+If you're looking for the <abbr title="To Long; Didn't Read">tl;dr</abbr> guide to getting started with the docs, this is it.
+
+1. Either install [the AsciiDoc Live Preview plugin](https://asciidoctor.org/docs/editing-asciidoc-with-live-preview/) for Firefox, Google Chrome, or Opera or [an AsciiDoc Live Preview plugin](https://asciidoctor.org/docs/editing-asciidoc-with-live-preview/#using-a-modern-text-editoride), if your text editor or IDE has one.
+2. In a feature branch, branched from master, change the relevant [AsciiDoc](./docs/what-is-asciidoc.md) files.
+3. Push your feature branch to the repository and create a PR from it.
+4. Make any requested changes.
+5. Your PR will be merged.
+
 ## Contributing to the Docs
 
-If you've been contributing to the previous version of ownCloud's documentation, which used reStructuredText and Sphinx-Doc, here’s how to get started contributing to the new version of the documentation.
+If you've been contributing to the previous version of ownCloud's documentation, which used reStructuredText and Sphinx-Doc, here's how to get started contributing to the new version of the documentation.
+Unlike the previous version of the documentation, you don't, necessarily, need to install a large number of tools and packages just to review your changes.
+
+### Only Making Text Changes?
+
+If you're only making text changes, then install either:
+
+1. The [AsciiDoc Live Preview plugin](https://asciidoctor.org/docs/editing-asciidoc-with-live-preview/) for *Firefox*, *Google Chrome*, or *Opera*
+2. The [AsciiDoc Live Preview plugin](https://asciidoctor.org/docs/editing-asciidoc-with-live-preview/#using-a-modern-text-editoride) for your text editor or IDE, *if it has one*.
+
+Using one, or both, of these, you can quickly check if the changes you make are what you expect, and if there are any render errors.
+If the document renders as you expect, then you can commit the changes and push them to the docs repository.
+
+### Are You Working With Inline Code Examples, Images, and Attachments?
+
+If, however, you're working with inline code examples, images, and attachments, then you need to [install Antora's command-line tools](./docs/install-antora.md).
```

Comment:
Is it true for inline code examples too?
I thought it is only true when you link to things outside the .adoc file

Reply:
It also makes sense for code examples to see that it's rendered like expected.

Reply:
@tboerger maybe misunderstood, imho rendering code examples would work with the minimal setup described above, as long they are in the text and not linked, so the need for the full installation of antora to render code examples is what I doubt as prerequisit...

Reply:
I think that I wasn't the clearest in what I said (I'll correct it). You can have code examples that directly contain the code, such as:

```asciidoc
.app.rb
[source,ruby]
----
require 'sinatra'

get '/hi' do
  "Hello World!"
end
----
```

And you can have them that have code sourced from a file, such as:

```asciidoc
[source,ruby]
----
include::app.rb[]
----
```

It's the second type that I'm referring to.